README.md

```diff
@@ -97,7 +100,7 @@ Please navigate [here](https://github.com/EpocDotFr/webtodotxt/blob/master/api.m
 Instead, use native mobile apps to edit and sync the Todo.txt file:
 
 **On Android**, you can use [Simpletask](https://play.google.com/store/apps/details?id=nl.mpcjanssen.todotxtholo&hl=en)
-(free) which can natively sync your tasks with your Dropbox. If you're using another storage provider (third-party or
+(free) which can natively sync your tasks with your Dropbox and Nextcloud. If you're using another storage provider (third-party or
```

Comment:
Simpletaks doesn't support Nextcloud.

Reply:
There is a version that does: https://play.google.com/store/apps/details?id=nl.mpcjanssen.simpletask.nextcloud

That's the main reason I made these efforts ;)

Reply:
Oh, NVM then. Can you add a link to this version?